content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/425-nextjs-prisma-client-monorepo.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaTitle: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaDescription: 'Using Prisma Client in a Next.js project in a monorepo setup'
+---
+
+## Problem
+
+If you use Prisma Client in a [Next.js](https://nextjs.org/) application within a monorepo, you will likely run into an error such as the following:
+
+```
+Error: Your schema.prisma could not be found, and we detected that you are using Next.js.
+Find out why and learn how to fix this: https://pris.ly/d/schema-not-found-nextjs
+```
+
+If you are using a Prisma version earlier than 4.11.0, the error message you will recieve will look similar to the message below:
+
+```
+Error: ENOENT: no such file or directory, open '/prisma-next-mono-reproduction/packages/service/.next/cache/webpack/client-development/schema.prisma'
+```
+
+Assume you have a monorepo with the following structure:
+
+```
+.
+├── packages
+│   ├── db
+│   │   ├── index.ts
+│   │   ├── node_modules
+│   │   ├── package.json
+│   │   └── prisma
+│   │       ├── client // <-- Custom output location for the generated Prisma Client
+│   │       │   ├── index.js
+│   │       │   ├── libquery_engine-debian-openssl-1.1.x.so.node // engine to be copied
+│   │       │   └── schema.prisma // schema to be copied
+│   │       └── schema.prisma
+│   └── service/
+│       ├── pages/
+│       │   └── api/
+│       │       └── test.js
+│       ├── next.config.js
+│       └── package.json
+├── pnpm-workspaces.yaml
+├── package.json
+└── vercel.json
+```
+
+The error mentioned above occurs as a result of a bundling problem during Next.js's bundling process. The `schema.prisma` and Query Engine files are expected to be found next to the generated Prisma Client. The bundling process, however, does not copy over those files to the output location of the bundle.
+
+This causes an issue as Prisma Client expects `schema.prisma` and the Query Engine to still be available next to Prisma Client within the bundled output. The Prisma schema and Query Engine were not copied over so in the context of the bundled output location, `schema.prisma` is nowhere to be found.
```

Comment:
First sentence here is a repetition of the previous paragraph.
The second sentence as well really.